scripts/ci/check_compliance.py

```diff
@@ -1244,6 +1244,7 @@ def check_no_undef_outside_kconfig(self, kconf):
         "SHIFT",
         "SINGLE_APPLICATION_SLOT", # Used in sysbuild for MCUboot configuration
         "SINGLE_APPLICATION_SLOT_RAM_LOAD", # Used in sysbuild for MCUboot configuration
+        "SKIP_ZERO_DUTY_CYCLE_TEST",
```

Comment:
Since there is a Kconfig file defining the option, I don't think you need to add it to this allowed list.